app/App.dev.js

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
 import { StyleSheet, Text, View } from 'react-native';
-import { SplashScreen } from 'expo';
+//import * as SplashScreen from 'expo-splash-screen';
 import { registerRootComponent } from 'expo';
 
 import { CodeGalaxyApp } from './target/scala-2.13/scalajs-bundler/main/codegalaxy-app-fastopt';
 
-SplashScreen.preventAutoHide()
+//SplashScreen.preventAutoHideAsync()
```

Comment:
Still needed?

Reply:
It has some issue atm. but, yes, we would have to enable it once the issue is solved. Because Splash screen is showing, but it hides too early atm. before the full load of the app (all the resources and auto-login of the user), so there is small blank page shown shortly.